### DISCUSSION TITLE
Mouse frame selection is configurable

### DISCUSSION THREAD

**Discussion by zyh374892663** [xyflow/xyflow#2231] [New Features]
Most users of the company's products are heavy users of `figma`, and they expect to drag the canvas through the space + left button, and realize frame selection directly through the left button of the mouse. At present, the space + left button can drag the canvas, but it is not possible to modify the left mouse button to select the box through configuration.

**Comment by moklick** [comment]
Hey. Is this what you are looking for https://github.com/wbkd/react-flow/discussions/2167 ?
